### PR TITLE
feat(condo): INFRA-641 optimize queries - part 2 (migrate tasks to new client-utils)

### PR DESCRIPTION
**Pull request by Alllex202**
- [x] Migrated tasks to new client-utils
- [x] Refactored tasks logic
- [x] Optimized queries
- [x] Added types
- [x] Removed feature flag
- [x] Updated callcenter submodule
- [x] Updated pass submodule

### CODE REVIEW

File: apps/condo/domains/common/components/tasks/storage/TasksCondoStorage.tsx
Comment (SavelevMatthew) [inline apps/condo/domains/common/components/tasks/storage/TasksCondoStorage.tsx:1]: What is condo storage?

File: apps/condo/domains/banking/queries/BankSyncTask.graphql [hunk 1]
```diff
@@ -0,0 +1,69 @@
+query getBankSyncTasks ($where: BankSyncTaskWhereInput!) {
```
Comment (SavelevMatthew) [inline apps/condo/domains/banking/queries/BankSyncTask.graphql:1]: Why do you need such queries? Seems like you have common fetched below....

And why raw where is passed? Is it possible to be more specific?

File: apps/condo/domains/ticket/queries/IncidentExportTask.graphql [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            publicUrl
+            originalFilename
+        }
+        totalRecordsCount
+        exportedRecordsCount
+        __typename
+    }
+}
+
+mutation updateIncidentExportTask ($data: IncidentExportTaskUpdateInput!, $id: ID!) {
```
Comment (SavelevMatthew) [inline apps/condo/domains/ticket/queries/IncidentExportTask.graphql:33]: why id is second argument? I thought all our API is designed to pass ID as first arg (what to change), and data is after (what exactly to change)

File: apps/condo/domains/common/components/tasks/storage/TasksCondoStorage.tsx
Comment (Alllex202) [reply apps/condo/domains/common/components/tasks/storage/TasksCondoStorage.tsx:1]: condo tasks are our deferred tasks that we create via the condo api

local tasks are tasks that are not related to the condo api (for example, tasks from miniapps)

Comment (Alllex202) [reply apps/condo/domains/common/components/tasks/storage/TasksCondoStorage.tsx:1]: I wouldn't like to change the names in this pr

File: apps/condo/domains/common/components/tasks/index.tsx [hunk 3]
```diff
@@ -47,48 +39,51 @@ export type TaskRecord = {
     taskId?: string
     sender?: string
 
-    createdAt: string
+    createdAt?: string
 }
+export type LocalTaskRecord = BaseTaskRecord & Required<Pick<BaseTaskRecord, 'user' | 'progress' | 'status' | 'createdAt'>>
+export type CondoTaskRecord = Pick<BaseTaskRecord, 'progress' | 'sender' | 'status' | '__typename' | 'id'>
```
[the remark this answers is not in the document]
Comment (Alllex202) [reply apps/condo/domains/common/components/tasks/index.tsx:45]: condo tasks are our deferred tasks that we create via the condo api

local tasks are tasks that are not related to the condo api (for example, tasks from miniapps)

I wouldn't like to change the names in this pr

**Review by nomerdvadcatpyat — APPROVED** [review]

**Review by SavelevMatthew — APPROVED** [review]

**Comment by Alllex202** [comment]
Now all updates will be in https://github.com/open-condo-software/condo/pull/5508.

**Pull request closed without merging**